main.cpp

```diff
@@ -0,0 +1,228 @@
+// Configuration editor
+// Command line utiltiy to create configuration files
+// Valid arguments are init and edit
+
+#include <iostream>
+#include <string>
+#include <fstream>
+#include <stdio.h>
+
+
+using std::cout;
+using std::endl;
+using std::string;
+using std::cin;
+
+const string CONFIG_FILE_PATH = "/users/bconway/SourceTree/School/CPT-180-27-Assignment-2-Config-Generator/app.cfg";
+
+
+int main(int argc, char *argv[]) {
+
+    string argument1 = "";
+    // Make sure that the argument isn't empty
+    if (argv[1] != nullptr) {
```

Comment:
I was having an issue getting my program to run without an argv[2]. Using this same kind of if statement with argv[2] helped fix that, so I have to say, thank you!! Following through your code helped me fix that.